src/main.rs

```diff
@@ -412,3 +416,33 @@ fn extract_frontary_keys_from_file(path: &Path, re: &Regex) -> Result<HashSet<St
 
     Ok(keys)
 }
+
+fn compare_keys(name1: &str, set1: &HashSet<String>, name2: &str, set2: &HashSet<String>) {
+    println!("=== {name1} vs {name2} ===");
+
+    let missing_in_2 = set1.difference(set2).fold(String::new(), |mut acc, key| {
```

Comment:
Please extract the duplicated code within the function into a separate function.